spark-extension-shims-spark3/src/test/scala/org/apache/spark/sql/blaze/BlazeFunctionSuite.scala

```diff
@@ -21,12 +21,9 @@ class BlazeFunctionSuite extends org.apache.spark.sql.QueryTest with BaseBlazeSQ
 
   test("sum function with float input") {
     withTable("t1") {
-      withSQLConf("spark.blaze.enable" -> "false") {
-        sql("set spark.blaze.enable=false")
-        sql("create table t1 using parquet as select 1.0f as c1")
-        val df = sql("select sum(c1) from t1")
-        checkAnswer(df, Seq(Row(1.23, 1.1)))
-      }
+      sql("create table t1 using parquet as select 1.0f as c1")
+      val df = sql("select sum(c1) from t1")
+      checkAnswer(df, Seq(Row(1.0)))
```

Comment:
it would fail with spark-3.0

Reply:
Thanks, I see this test case has been fixed in #1161, I will close this PR.